test/UtilsTest.js

```diff
@@ -256,6 +256,31 @@ describe('FlowLimit', async () => {
     });
 });
 
+describe('ChainTracker', async () => {
+    let test;
+    const chainName = 'Chain Name';
+
+    before(async () => {
+        test = await deployContract(ownerWallet, 'TestChainTracker');
+    });
+
+    it('Should calculate hardcoded constants correctly', async () => {
+        await expect(deployContract(ownerWallet, `TestChainTracker`, [])).to.not.be.reverted;
+    });
+
+    it('Should set remove and query a chain properly', async () => {
+        expect(await test.isTrustedChain(chainName)).to.equal(false);
+
+        await expect(test.setTrustedChain(chainName)).to.emit(test, 'TrustedChainSet').withArgs(chainName);
+
+        expect(await test.isTrustedChain(chainName)).to.equal(true);
+
+        await expect(test.removeTrustedChain(chainName)).to.emit(test, 'TrustedChainRemoved').withArgs(chainName);
+
+        expect(await test.isTrustedChain(chainName)).to.equal(false);
+    });
```

Comment:
this should be tested on the ITS contract directly

Reply:
it is tested in this section: https://github.com/axelarnetwork/interchain-token-service/blob/feat/only-hub-chains/test/InterchainTokenService.js#L697